### DISCUSSION TITLE
<Link /> element does not work in both .md and .mdx file

### DISCUSSION THREAD

**Discussion by piavgh** [facebook/docusaurus#4529] [Q&A]
```
---
title: Pool Addresses
sidebar: Docs
---

# Overview

lorem ipsum...

# getPools

The best way to get all pools for a specific token pair is to call <Link to='/smart-contracts/factory/#getpools'>getPools</Link> on the factory, then selecting one of the pools from the returned array. If none exists, an empty array is returned.
```

The `<Link to='/smart-contracts/factory/#getpools'>getPools</Link>` does not work as expected. It looks like normal text:

![Screenshot at Mar 29 10-59-47](https://user-images.githubusercontent.com/4060207/112785320-ed364280-907d-11eb-8815-f421e87ffbb2.png)

How can I fix this issue?
Thank you.

**Comment by slorber** [comment]
First, why don't you use a regular markdown link?

2nd, it works for me:

![image](https://user-images.githubusercontent.com/749374/112855149-d65c1400-90ae-11eb-8e77-b86e86134b9d.png)

![image](https://user-images.githubusercontent.com/749374/112855103-ce03d900-90ae-11eb-8047-c0aa2c179c96.png)

**Reply by piavgh** [reply]
oh, so we have to `import Link` explicitly like that? :D 

I used to work with Gatsby and I thought that it should import implicitly.

Thank you

**Reply by slorber** [reply]
I plan to make it so the import is not necessary, but yes by default you have to import comps used in MDX.

In the end it's compiled to JSX => https://mdxjs.com/playground/

**Reply by piavgh** [reply]
ok, thanks again
